new/LIT7240PPAddam.xml

```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7240PPAddam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Prayer by virtue of the secret names revealed by God to Adam</title>
+                <title xml:lang="gez" xml:id="t2">ዘለሐከ፡ ለአዳም፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Za-laḥaka la-ʾAddām</title>
```

Comment:
I think it might be related to laḥakʷa "who formed Adam taking soil"
https://betamasaheft.eu/Dillmann/lemma/Lc3b6a912d7774d19b6953db0d4affea3